Tests/MapboxMapsTests/Foundation/MapboxMapsFoundationTests.swift

```diff
@@ -73,36 +73,36 @@ class MapboxMapsFoundationTests: XCTestCase {
         // Convert the subview's center to a coordinate.
         // The subview's center is expected to be at the center coordinate of the map.
         let center = CGPoint(x: subView.bounds.midX, y: subView.bounds.midY)
-        let coordinate = mapView.coordinate(for: center, in: subView)
+        let coordinate = mapView.mapboxMap.coordinate(for: center)
 
         XCTAssertEqual(coordinate.latitude, CLLocationDegrees(0), accuracy: accuracy)
         XCTAssertEqual(coordinate.longitude, CLLocationDegrees(0), accuracy: accuracy)
     }
 
     func testPointToCoordinateInSubViewEqualOrigins() {
-        let subViewRect = CGRect(x: 0,
-                                 y: 0,
-                                 width: mapView.bounds.size.width / 2,
-                                 height: mapView.bounds.size.height / 2)
-        let subview = UIView(frame: subViewRect)
-
-        mapView.addSubview(subview)
-
-        /**
-         We shouldn't expect the centers to be the same, since now that the
-         mapView is offset. The "center" is in the space of the parent view.
-         */
-        subview.center = CGPoint(x: mapView.bounds.midX, y: mapView.bounds.midY)
-        XCTAssertNotEqual(subview.center, mapView.center, "Center of both views are not equal")
-        XCTAssertEqual(subview.frame.origin.x, 25.0)
-        XCTAssertEqual(subview.frame.origin.y, 25.0)
-
-        let updatedSubViewOrigin = subview.frame.origin
-        let originCoordinateA = mapView.coordinate(for: updatedSubViewOrigin, in: mapView)
-        let originCoordinateB = mapView.coordinate(for: CGPoint.zero, in: subview)
-
-        XCTAssertEqual(originCoordinateA.latitude, originCoordinateB.latitude, accuracy: accuracy)
-        XCTAssertEqual(originCoordinateA.longitude, originCoordinateB.longitude, accuracy: accuracy)
+//        let subViewRect = CGRect(x: 0,
```

Comment:
Why commented out?

Reply:
I'm not sure if this is a relevant test anymore..